qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -612,7 +620,7 @@ def acquire(self, mode=None, samples_per_record=None,
                 alloc_buffers=None, fifo_only_streaming=None,
                 interleave_samples=None, get_processed_data=None,
                 allocated_buffers=None, buffer_timeout=None,
-                acquisition_controller=None):
+                acquisition_controller=None) -> np.ndarray:
```

Comment:
The acquire method can return anything. acquisition controller's post_acquire defines this.
```suggestion
                acquisition_controller=None):
```

Reply:
sure! but it's just another one ~~brick in the wall~~ thing to do with the core Alazar driver among tons of little things :)